app/assets/stylesheets/arctic_admin/pages/_form.scss

```diff
@@ -1,5 +1,5 @@
 body.logged_in {
-  &.new, &.edit, &.create, &.update {
+  &.new, &.edit, &.create, &.update, &.show {
```

Comment:
I'm afraid for this because you can have a `.actions` class in your show page and this will apply : 

```css
    .actions {
      padding-left: 25%;
    }
```

Reply:
Oh, I see :)
This pr should be closed!